infrastructure/smart-contracts/test/FixtureAuditTest.mjs

```diff
@@ -0,0 +1,444 @@
+import { parse } from "csv/sync";
+import fs from "fs";
+import { config, createBlsKeyPair } from "@sentry/core";
+import { extractAbi } from "../utils/exportAbi.mjs";
+import { RefereeBulkSubmissionTransfers } from "./tinykeys/RefereeBulkSubmissionTransfers.mjs";
+import { mintBatchedLicenses } from "./utils/mintLicenses.mjs"
+
+describe("Fixture Tests", function () {
```

Comment:
Please rename, does not make sense to have two Fixtures with the exact name running different tests.

Reply:
Done!